backend/src/main/java/com/eshka/entity/Team.java

```diff
@@ -1,14 +1,23 @@
 package com.eshka.entity;
 
 import lombok.Data;
+import lombok.Builder;
 import lombok.NoArgsConstructor;
 import lombok.AllArgsConstructor;
 
+import javax.persistence.*;
+
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
+@Builder
+@Entity(name = "Team")
 public class Team {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
     private long id;
+    @Column(name = "name", nullable = false)
     private String name;
+    @Column(name = "description", unique = true, nullable = false)
```

Comment:
description of team must be unique when name not?

Reply:
No, the description can by any.